src/sentry/receivers/outbox.py

```diff
@@ -46,7 +47,7 @@ def _maybe_process_tombstone(model: Type[T], object_identifier: int) -> T | None
     return None
 
 
-@receiver(process_region_outbox, sender=OutboxCategory.USER_UPDATE)
+@receiver(process_control_outbox, sender=OutboxCategory.USER_UPDATE)
```

Comment:
Should we consider having different files/modules for outbox messages that are processed on control vs. region? Having separate modules might help with reading comprehension.

Reply:
That's totally fair.  I'll see if I can divide this module up some.